gto-support-gson/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest package="org.ccci.gto.android.common.gson"/>
```

Comment:
yeah, it's still a library project which requires a bundled manifest. Any resources defined in a library project get merged into the main project manifest, but I've only used that in a couple places before.